content/id/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,221 @@
+2017/03/21 06:48:17 [INFO] encoded CSR
+```
+
+Perintah ini menghasilkan dua berkas; Ini menghasilkan `server.csr` yang berisi permintaan sertifikasi PEM
+tersandi [pkcs#10](https://tools.ietf.org/html/rfc2986),
+dan `server-key.pem` yang berisi PEM kunci yang tersandi untuk sertifikat yang
+masih harus dibuat.
+
+## Membuat objek _Certificate Signing Request_ untuk dikirim ke API Kubernetes
+Buat sebuah yaml CSR dan kirim ke ApiServer dengan menggunakan perintah berikut:
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: certificates.k8s.io/v1beta1
+kind: CertificateSigningRequest
+metadata:
+  name: my-svc.my-namespace
+spec:
+  request: $(cat server.csr | base64 | tr -d '\n')
+  usages:
+  - digital signature
+  - key encipherment
+  - server auth
+EOF
+```
+
+Perhatikan bahwa berkas `server.csr` yang dibuat pada langkah 1 merupakan base64 tersandi
+dan disimpan di _field_ `.spec.request`. Kami juga meminta
+sertifikat dengan penggunaan kunci "_digital signature_", "_key enchiperment_", dan "_server
+auth_". Kami mendukung semua penggunaan kunci dan penggunaan kunci yang diperpanjang yang terdaftar
+[di sini](https://godoc.org/k8s.io/api/certificates/v1beta1#KeyUsage)
+sehingga Kamu dapat meminta sertifikat klien dan sertifikat lain menggunakan
+API yang sama.
+
+CSR semestinya bisa dilihat dari API pada status _Pending_. Kamu bisa melihatnya dengan menjalankan:
+
+```shell
+kubectl describe csr my-svc.my-namespace
+```
+
+```none
+Name:                   my-svc.my-namespace
+Labels:                 <none>
+Annotations:            <none>
+CreationTimestamp:      Tue, 21 Mar 2017 07:03:51 -0700
+Requesting User:        yourname@example.com
+Status:                 Pending
+Subject:
+        Common Name:    my-svc.my-namespace.svc.cluster.local
+        Serial Number:
+Subject Alternative Names:
+        DNS Names:      my-svc.my-namespace.svc.cluster.local
+        IP Addresses:   192.0.2.24
+                        10.0.34.2
+Events: <none>
+```
+
+## Mendapatkan Persetujuan _Certificate Signing Request_
+
+Penyetujuan _Certificate Signing Request_ dapat dilakukan dengan otomatis
+atau dilakukan sekali oleh administrator klaster. Informasi lebih lanjut tentang 
+apa yang terjadi dibahas dibawah ini.
+
+## Unduh dan Gunakan Sertifikat
+
+Setelah CSR ditandatangani dan disetujui, Kamu akan melihat:
+
+```shell
+kubectl get csr
+```
+
+```none
+NAME                  AGE       REQUESTOR               CONDITION
+my-svc.my-namespace   10m       yourname@example.com    Approved,Issued
+```
+
+Kamu bisa mengundur sertifikat yang telah diterbitkan dan menyimpannya ke berkas 
+`server.crt` dengan menggunakan perintah berikut:
+
+```shell
+kubectl get csr my-svc.my-namespace -o jsonpath='{.status.certificate}' \
+    | base64 --decode > server.crt
+```
+
+Sekarang kamu bisa menggunakan `server.crt` dan `server-key.pem` sebagai pasangan 
+kunci untuk memulai server HTTPS Kamu.
+
+## Penyetujuan _Certificate Signing Requests_
```

Comment:
nit: in other places this is written in singular, let's make it consistent
```suggestion
## Penyetujuan _Certificate Signing Request_
```

Reply:
actually, CertificateSigningRequest itself is an API object, can we change accordingly here in this entire page?
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#certificatesigningrequest-v1beta1-certificates-k8s-io

```suggestion
## Penyetujuan CertificateSigningRequest
```

I won't give same comments for the rest of lines